packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -129,6 +314,8 @@ class CupertinoContextMenu extends StatefulWidget {
   /// This parameter cannot be null or empty.
   final List<Widget> actions;
 
+  /// Deprecated in favor of using [builder] for more control.
+  ///
```

Comment:
I think there's no need to add this since you already have the `@Deprecated` tag below.  If I'm wrong and you see this pattern elsewhere then nevermind!